server/src/bootstrap/config/database.ts

```diff
@@ -44,6 +48,30 @@ const dbSchema: Schema<DbConfig> = {
     format: String,
     default: 'mysql',
   },
+  searchHost: {
+    doc: 'Host for opensearch service',
+    format: String,
+    default: 'localhost',
+    env: 'SEARCH_HOST',
+  },
+  searchUsername: {
+    doc: 'Username for opensearch service',
+    format: String,
+    default: 'admin',
+    env: 'SEARCH_USERNAME',
+  },
+  searchPassword: {
+    doc: 'Password for opensearch service',
+    format: String,
+    default: 'admin',
+    env: 'SEARCH_PASSWORD',
+  },
+  searchPort: {
+    doc: 'Port for opensearch service',
+    format: 'port',
+    default: 9200,
+    env: 'SEARCH_POST',
+  },
```

Comment:
er.... no... move this to its own `config/search`, and drop the `searchXXX` prefix in favour of just `xxx`

Reply:
Ohh okay oops sorry